frontend/src/components/Psychologist/Profile/ConventionForm.jsx

```diff
@@ -35,6 +35,9 @@ const ConventionForm = ({ currentConvention, onConventionUpdated, checkDefaultVa
       });
   };
 
+  const laSorbonne = 'b394d7e8-a968-53f7-9016-e37d070080b0';
+  const laSorbonneParisNord = '47ec8346-6019-588e-af40-2c437361ecac';
+
```

Comment:
that's not the prod uuid...
Would be nice to get all the universities  from the back and find the one by name ?

Reply:
:thinking: why not compare by name?

Reply:
that's what he did ?

Reply:
Yep i did it already

Reply:
Oops! I didn't understand it right, sorry!